doc(var) load default var indexes (#35)
Kong/lua-kong-nginx-module: README.md

- `$http_user_agent`
- `$host`

> If the parameter is `default`, then this directive will try to
> load *commonly used variables*.
> 
> Here is the list of indexed variables by `default`:

Reviewer: ```suggestion
Specially, use `lua_kong_load_var_index default` to
index *commonly used variables* as follows:
```
